vcd/sample_vcd_test_config.json

```diff
@@ -72,5 +72,12 @@
     "//": "Size in megabytes",
     "uploadPieceSize": 5,
     "uploadProgress": true
+  },
+  "envVariables" : {
+    "//" : "Environment variables that we want to set (or unset)",
```

Comment:
It's fully clear, I suggest rephrasing to: "Environment variables that we want the test to set (or unset)"

Reply:
Done